src/docs/GetStarted/Designers/namingConvention/index.mdx

```diff
@@ -1,8 +1,40 @@
 ---
-title: "Naming convention"
+title: 'Naming convention'
 order: 5
 ---
 
-#About the naming convention
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
```

Comment:
maybe have a different intro for consumers than for contributors, telling them that those naming conventions are here to help them identify quicker the right element for their need.

Reply:
this is mainly for contributors as far as I understood. we won't check if "consumers" are using it in this way. we'll not impose that. but for `contributors` its the goal.